.github/workflows/labels.yml

```diff
@@ -212,6 +212,11 @@ jobs:
                   if (approvals.size > 0) after.push(`12.approvals: ${approvals.size > 2 ? '3+' : approvals.size}`)
                   if (Array.from(maintainers).some(m => approvals.has(m))) after.push('12.approved-by: package-maintainer')
 
+                  if (context.eventName == 'pull_request') {
+                    core.info('Skipping labeling on a pull_request event (no privileges).')
+                    return
+                  }
```

Comment:
Ah I see, so we skip the `actions/labeller` steps on non-`pull_request_target` events, but for this step we run as far as we can before terminating, because we have control.